### PR TITLE
react app setup

### PR DESCRIPTION
**Pull request by mikhail-balabaev**
[AB#2307](https://deloittedigitalat.visualstudio.com/38398c5c-11b2-4e64-98b5-5af5e65d6e82/_workitems/edit/2307)

- Created React app with routing & typescript.
- Added Lint and Prettier
- Add pre-commit hook

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "frontend",
```
Comment (aberke220) [inline frontend/package.json:2]: Would it be possible to call it `credit-application-frontend` to make in consistent with the documentation?

**Review by aberke220 — APPROVED** [review]
Approved + question about project naming.

**Comment by theqajedi** [comment]
will have to create a new CI for this project.

File: frontend/package.json (hunk 1 above)
Comment (mikhail-balabaev) [reply frontend/package.json]: done

**Pull request merged**